controllers/release/release_adapter_test.go

```diff
@@ -717,13 +742,22 @@ var _ = Describe("Release Adapter", Ordered, func() {
 		// WithReleaseStrategy()
 		Expect(pipelineRun.Spec.PipelineRef.Name).To(Equal(releaseStrategy.Spec.Pipeline))
 		Expect(pipelineRun.Spec.PipelineRef.Bundle).To(Equal(releaseStrategy.Spec.Bundle))
+
 		Expect(pipelineRun.Spec.Params[0].Name).To(Equal("policy"))
```

Comment:
Why is this here? We should not be passing the `policy` field any more, just the JSON representation on a `enterprisecontractpolicy` parameter.

In addition, we should not use `Params[n]`. If we change the order in the function itself, the tests would fail. Would be better to check if there's a Param containing the given values. Maybe it would make sense to change it here.

Reply:
@davidmogar  `pipelineRun.Spec.Params[0]` is the `releaseStrategy` not the just added EC Policy which is the `pipelinerun.Spec.Params[1]`. 
I agree. I used the way it was before to not refactory the tests but I can do it NP.

Reply:
We should not pass any `policy` as a parameter. Just the spec. That's the change you are working on here and it's actually missing. That should be removed in the test and in the code.

Reply:
I did not. You are looking to the wrong place. What you are looking for is a bit down the code. The code you pointed has nothing to do with the new feature, it regards the releaseStrategy. It was already there, but with different index. 
The test is at line `750`. This one:

```
		// WithEnterpriseContractPolicy()
		enterpriseContractPolicy, err := adapter.getEnterpriseContractPolicy(releaseStrategy)
		Expect(err).ShouldNot(HaveOccurred())

		enterpriseContractPolicyJsonSpec, _ := json.Marshal(enterpriseContractPolicy.Spec)
		Expect(pipelineRun.Spec.Params[1].Name).To(Equal("enterprisecontractpolicy"))
		Expect(pipelineRun.Spec.Params[1].Value.StringVal).To(Equal(string(enterpriseContractPolicyJsonSpec)))
``` 

That said, I am working on refactoring this full test.

Reply:
@davidmogar I refactored this full test. Please take a look.